azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -230,12 +239,14 @@ def create_from_edge_environment(cls):
                 new_err = OSError("Unexpected failure in IoTEdge")
                 new_err.__cause__ = e
                 raise new_err
-        iothub_pipeline = pipeline.IoTHubPipeline(authentication_provider)
+
+        pipelineConfiguration = config.BasePipelineConfig(**kwargs)
```

Comment:
>pipelineConfiguration [](start = 8, length = 21)

Here and everywhere - camelCase indicates a class definition in Python. Please always use snake_case for everything else.

Reply:
fixed.